### DISCUSSION TITLE
"Unhandled error" message appears everytime I open the game.

### DISCUSSION THREAD

**Discussion by insecurestudios** [ppy/osu#27939] [Q&A] [answered]
Whatever the error is does not seem to affect gameplay, but the message happening every time is annoying. I am on Linux for chromebook with the latest flatpak version. This message only started popping up this update, it was not doing this before.
[compressed-logs.zip](https://github.com/ppy/osu/files/15046541/compressed-logs.zip)

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/flathub/sh.ppy.osu/issues/170 and fixed with a workaround(?) in https://github.com/ppy/osu/pull/27908.
